src/language/LanguageProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { createContext, useContext, useEffect } from "react";
+import sv from "language/sv.json";
+
+import languages from 'language';
+
+import Storage from 'storage';
+
+const LanguageContext = createContext(sv);
+
+/**
+ * Provides the current language dictionary to its children.
+ */
+export const LanguageProvider: React.FC = ({children}) => {
+  const [settings, modifySettings] = Storage.useSettings();
+  const dict = languages[settings.language];
+
+  return (
+    <LanguageContext.Provider value={dict}>
+      {children}
+    </LanguageContext.Provider>
+  );
+};
+
+/**
+ * Function returning the current language dictionary.
+ * 
+ * Language is changed from the useSettings hook.
+ * 
+ * @example 
+ * const dict = useTranslation();
+ * dict["valuesHeaderEvaluation"];
+ * ```
+ * returns "Värdering" if swedish is the current language.
+ * 
+ * keyvalues can be found in the respective json, and new translations should
+ * be added in every file (src/language/??.json).
+ * ```
```

Comment:
We did it this way to highlight the text correctly when hovering over the function with vs code.

![unknown](https://user-images.githubusercontent.com/69213759/114189606-aaf8e500-994a-11eb-8d9d-91e19a6ff812.png)

Reply:
That is wierd. It should highlight correctly by following the [TSDoc standard](https://tsdoc.org/pages/tags/example/) I would think. Perhaps try adding "ts" right after the ``` and see if it renders correctly?

Like so:
```ts